source/stylesheets/site/_layout.scss

```diff
@@ -0,0 +1,26 @@
+html,
+body {
+  height: initial;
+}
+
+body {
+  @media (min-width: $medium-screen) {
+    background-image: linear-gradient($danube, $french-pass);
+    padding: $small-spacing;
+  }
+
+  @media (min-width: 80em) {
```

Comment:
I’m seeing some pixel-based queries, and some em-based. Is that by design? Should we unify, if not?